helpers/control.py

```diff
@@ -374,11 +378,12 @@ def inject_prepare(self):
             self.node.run_cmd('mkdir %s' % self.modules_dir)
             cmd = 'find %s/ -name "*.ko" | xargs cp -t %s'
             self.node.run_cmd(cmd % (self.workdir, self.modules_dir))
+            self.node.copy_file_to_node(self.stap_local, '/tmp/')
```

Comment:
Each node have a `workdir` variable. This directory is usually used for storing generated files or other files. In the case of `tempesta` node it points to sources directory.  It's not good to put some files used for tests purposes into some random directories and define that directories all across tests suite. The directory is defined in the tests suite config and no other directories must be used.

There one more point. `/tmp/` is usually ramdisk and not persistent storage and 'cleaned' on reboot. It's good: after reboot you get completely clean test environment. But some times it's not the desired behaviour, if Tempesta crashes on the tests you may like to have previous configuration files for a new manual try. Thus the directory must be defined in config file.

Reply:
I've selected /tmp, because tempesta config is also stored there

Reply:
It's set to `tmp` by default, variable `config` for `tempesta` node in `tests_config.ini` controls config file location. There can be sense in renaming `tempesta.workdir` -> `tempesta.srcdir` and `tempesta.config` -> `tempesta.workdir` to make `tempesta.workdir`  to have the same meaning as `workdir` for other nodes does.

Reply:
Yes, good idea